jcli/src/jcli_app/rest/mod.rs

```diff
@@ -10,8 +11,37 @@ pub enum Rest {
     V0(v0::V0),
 }
 
+const SERIALIZATION_ERROR_MSG: &'static str = "node returned malformed data";
```

Comment:
what is the rational for this const?

Reply:
The same issue can be manifested in two kinds of errors and this ensures that it is presented in consistent way to the user